central/detection/service/service_impl_test.go

```diff
@@ -685,3 +690,23 @@ func TestFetchOptionFromRequest(t *testing.T) {
 		})
 	}
 }
+
+func TestGetAppliedNetpolsForDeployment(t *testing.T) {
+	mockCtrl := gomock.NewController(t)
+	mockNetpolDS := networkPolicyMockStore.NewMockDataStore(mockCtrl)
+	mockNetpolDS.EXPECT().GetNetworkPolicies(gomock.Any(), gomock.Any(), gomock.Any()).Return(
```

Comment:
You know the cluster ID (2nd param) and namespace (3rd param), so maybe let's try to change `gomock.Any()` to something more concrete? 
In its current form, this mock will always return the contents of `Return(...)` block. If we specify those params, then the `Return(...)` block will be only returned when those parameters would match the call.
```suggestion
	clusterID := uuid.NewV4().String()
	mockNetpolDS.EXPECT().GetNetworkPolicies(gomock.Any(), clusterID, "ns").Return(
```

and then in 705 use the cluster id from the var `clusterID`.